sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,29 +1,41 @@
 # Release History
 
-## 2.0.0-beta.1 (Unreleased)
+## 2.0.0-beta.1 (2021-03-23)
 
 This release continues with the changes from `1.2.4` and `1.2.4-beta.1`.
 
+### Breaking changes
+
 - The `getToken` methods will now never return `null`. If a token is not available, we will return a rejected promise.
```

Comment:
This generally has no affect on the end user who we expect to just pass the credential. The main person who should update their code is Azure Core or any package that calls `getToken` directly. Did we update all those cases to adjust to this change?

Also, a small suggestion on re-wording the line here
```suggestion
-  If a token is not available, the promise returned by the `getToken` method on credentials will be rejected. Previously, the promise resolved with `null`.
```

Reply:
I'll add that suggestion separately to avoid resolving your comment.

TypeScript didn't have issues with this API change. Nothing broke. Not sure if I should go see if anything needs cleanup if the compiler doesn't complain 😅

Reply:
We should do more than just check that compiler doesnt complain. There might be code that checks for `null` case to determine whether there was a token or not. 

Also, in the identity.api.md file, I still see a lot of credential classes that have the getToken() method returning `Promise<AccessToken | null>`. Shouldnt that change?

Also, I recall some very old conversations around why we prefer returning null rather than throwing so that chaining becomes simpler. Is that no longer the case? Do we have language alignment here? Perhaps linking to the PR or issue that has more details will help

Reply:
I need to align these other PRs as well, that is true. I'll make an issue to align the `getToken` methods across the board.

Reply:
The | null call useful a while ago before we had the CredentialUnavailable error. Now it's not necessary and there's no logical path that results in a null token.

Reply:
Issue: https://github.com/Azure/azure-sdk-for-js/issues/14421

Reply:
I've made a PR to align the other credentials on Identity. The description of the PR states why I believe we don't need to update the rest of the codebase: https://github.com/Azure/azure-sdk-for-js/pull/14428